Update remove-former-employee-step-4.md (#5154)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/add-users/remove-former-employee-step-4.md

> You can move or copy up to 500 MB of files and folders at a time.<br/>
> When you move or copy documents that have version history, only the latest version is moved.  

> You can also grant access to another user to a former employee's OneDrive.

Reviewer: ```suggestion
You can also grant access to another user to access a former employee's OneDrive.
```

> You can move or copy up to 500 MB of files and folders at a time.<br/>
> When you move or copy documents that have version history, only the latest version is moved.  

> You can also grant access to another user on former employee's onedrive
> 

Reviewer: ```suggestion
You can also grant access to another user to a former employee's OneDrive.
```

1. Sign in to the <a href="https://go.microsoft.com/fwlink/p/?linkid=2024339" target="_blank">admin center</a> as a global admin or SharePoint admin.

    If you get a message that you don't have permission to access the admin center, then you don't have administrator permissions in your organization.

2. In the left pane, select **Admin centers** \> **SharePoint**. (You might need to select **Show all** to see the list of admin centers.)

3. If the classic SharePoint admin center appears, select **Open it now** at the top of the page to open the SharePoint admin center.

4. In the left pane, select **More features**.

5. Under **User profiles**, select **Open**.

6. Under **People**, select **Manage User Profiles**.

> 7. Enter the Former employee's name and select **Find**.

Reviewer: ```suggestion
7. Enter the former employee's name and select **Find**.
```

> 8. Right-click the user, and then choose **Manage site collection owners**.

Reviewer: ```suggestion
8. Right-click on the user, and then choose **Manage site collection owners**.
```

> 9. Add the **User** to the **Site collection Administrators** and Click On **Ok**

Reviewer: ```suggestion
9. Add the user to **Site collection administrators** and select **Ok**.
```

> 10. The **User** will now able to access the former employees Onedrive using the onedrive **URL** 

Reviewer: ```suggestion
10. The user will now be able to access the former employee's OneDrive using the OneDrive URL. 
```
